recipes/oic/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "oic" %}
+{% set version = "0.13.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e6cf115fa74c030c1fe4325d7ea980b00a1c492632cf1dad1aee93234810118a
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - alabaster
+    - beaker
+    - future
+    - mako
+    - pycryptodomex
+    - pyjwkest >=1.3.6
+    - pyopenssl
+    - python
+    - requests
+    - six
+
+test:
+  imports:
+    - oic
+    - oic.extension
+    - oic.oauth2
+    - oic.oic
+    - oic.utils
+    - oic.utils.authn
+    - oic.utils.rp
+    - oic.utils.userinfo
+  requires:
```

Comment:
Typically if you are only doing `imports` tests, you shouldn't need anything in `requires`. That section should only really be necessary if you're running a test suite that needs additional packages.